<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datacorn</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/res.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_4"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionNewDatabase"/>
    <addaction name="actionOpenDatabase"/>
    <addaction name="actionCloseDatabase"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHomepage"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionSpanish"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="textDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Text</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="descPlainTextEdit">
       <property name="placeholderText">
        <string>Card Text</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="typeDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Types</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="typesWidget"/>
  </widget>
  <widget class="QDockWidget" name="stringDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Strings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="stringsTableWidget">
       <property name="rowCount">
        <number>16</number>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>124</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="11" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="12" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="13" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="14" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="15" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="raceDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Races</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="racesWidget"/>
  </widget>
  <widget class="QDockWidget" name="linkMarkersDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>160</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Link Markers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="markerTopLeftButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>↖</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="markerTopButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>⬆</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="markerTopRightButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>↗</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="markerLeftButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>⬅</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="markerRightButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>➡</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="markerBottomLeftButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>↙</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="markerBottomButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>⬇</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="markerBottomRightButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>↘</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="categoryDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Categories</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="categoriesWidget"/>
  </widget>
  <widget class="QDockWidget" name="monsterStatDockWidget">
   <property name="windowTitle">
    <string>Monster Stats</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lScaleLabel">
         <property name="text">
          <string>L</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="lScaleSpinBox"/>
       </item>
       <item>
        <widget class="QSpinBox" name="rScaleSpinBox">
         <property name="suffix">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rScaleLabel">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="levelLabel">
         <property name="text">
          <string>Level/Rank</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="levelSpinBox">
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="attributeDockWidget">
   <property name="windowTitle">
    <string>Attributes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="attributesWidget"/>
  </widget>
  <widget class="QDockWidget" name="battleStatsDockWidget">
   <property name="windowTitle">
    <string>Battle Stats</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_10">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="atkLabel">
       <property name="text">
        <string>ATK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="atkQmCheckBox">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="atkSpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="defLabel">
       <property name="text">
        <string>DEF</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="defQmCheckBox">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="defSpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="codeDockWidget">
   <property name="windowTitle">
    <string>Codes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_11">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="passLabel">
       <property name="text">
        <string>pass</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="aliasLabel">
       <property name="text">
        <string>alias</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="passLineEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="aliasLineEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="scopeDockWidget">
   <property name="windowTitle">
    <string>Scope</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="scopesWidget"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Database</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_13">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Code</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewDatabase">
   <property name="text">
    <string>New Database...</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>Open Database...</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionCloseDatabase">
   <property name="text">
    <string>Close Database</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Español (Spanish)</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Inglés (English)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
